oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthorisationHandler.java

```diff
@@ -196,6 +200,17 @@ public APIGatewayProxyResponseEvent authoriseRequestHandler(
         attachLogFieldToLogs(CLIENT_SESSION_ID, clientSessionId);
         attachLogFieldToLogs(GOVUK_SIGNIN_JOURNEY_ID, clientSessionId);
 
+        try {
+            var txmaEncoded =
+                    RequestHeaderHelper.getHeaderValueFromHeaders(
+                            input.getHeaders(), "txma-audit-encoded", false);
+            if (txmaEncoded != null) {
+                attachAuditField(TXMA_ENCODED_HEADER, txmaEncoded);
+            }
+        } catch (InvalidEncodingException e) {
+            LOG.error(e.getMessage());
+        }
```

Comment:
Worth extracting this to a static helper method somewhere? We're going to be doing this a lot